### PR TITLE
Update ReadAndWrite.ino

### PR DESCRIPTION
**Pull request by ijash**
It's seems not that important, but this helps the new user understand the code.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Doc update?   | yes
| BC breaks?    | no 
| Deprecations? | no

### CODE REVIEW

**Comment by ijash** [comment]
i dont know what exactly `if (!mfrc522.PICC_ReadCardSerial())` did.

File: examples/ReadAndWrite/ReadAndWrite.ino [hunk 1]
```diff
@@ -64,7 +64,7 @@ void setup() {
  * Main loop.
  */
 void loop() {
-    // Look for new cards
+    // Reset the loop if no card present on the sensor/reader. This saves the entire process when idle.
```
Comment (Rotzbua) [inline examples/ReadAndWrite/ReadAndWrite.ino]: "new" is important so please change to:
`if no new cards`

**Comment by ijash** [comment]
I will, i'll edit it as you told me to.

**Comment by pug1-xda** [comment]
great to see development on this library... i have nearly finished my lock sketch and this is the base for the rfid part

**Pull request merged**